running/architecture.rst

```diff
@@ -2,3 +2,86 @@ Architecture
 ============
 
 Deconst is distributed as a set of Docker containers, deployed by an Ansible playbook.
+
+Terminology
+-----------
+
+.. glossary::
+
+  content repository
+    Location containing material to be included as a subset of the completed site. Often, the
+    material within will be written in a friendlier human-editable markup format such as
+    reStructuredText or Markdown.
+
+    Initially, we will support support content stored in git repositories on
+    `GitHub <github.com>`_, although our architecture will be flexible enough to integrate
+    content stored anywhere on a network reachable from the build system.
+
+  map repository
+    Version controlled repository containing plain-text documents that associate subtrees of
+    indexed content, identified by a :term:`content ID` prefix, with subtrees of :term:`paths`
+    on the presented site.
+
+  content ID
+    Unique identifier assigned to a single page of content generated from a :term:`content
+    repository`. It's important to note that a content ID is assigned to each *output* page, not
+    each source document. Depending on the :term:`preparer` and its configuration, these may differ.
+    Most of the architecture should treat these as opaque strings, although the :term:`mapping
+    service` may need to assume that they are hierarchal.
+
+    By convention, these are URLs that join the base URL of the :term:`content repository` with the
+    relative path of the rendered output page.
+
+    Examples:
+
+    * The version of this page on the default "master" branch:
+      ``https://github.com/deconst/deconst-docs/running/architecture``.
+    * The version of this page on a branch called "glossary":
+      ``https://github.com/deconst/deconst-docs/tree/glossary/running/architecture``.
+    * A specific post in a Jekyll blog, generated from (theoretical) content at
+      ``https://github.com/rackerlabs/developer-blog/_posts/mongodb-3.0-getting-started.md``:
+      ``https://github.com/rackerlabs/developer-blog/blog/mongodb-3.0-getting-started``.
```

Comment:
@kenperkins Is this any better? I'm struggling trying to communicate this one for some reason.